### ISSUE TITLE
After deleting the service, the user should remain on the service page

### ISSUE THREAD

**Issue by Marcelinna** [cyfronet-fid/whitelabel-marketplace#51]
Now :Delete service, User is redirected to /backoffice/services
Excepted behavior: User should stay on the service page, similar when changing status to unpublish, suspended

**Comment by Michal-Kolomanski** [comment]
I would argue that redirection to `/backoffice/services` on the delete action is the expected behaviour. 
Staying on the deleted service page will be weird from my perspective. 
@roksanaer @AlicjaSwierad @eldenkat any thoughts?

**Comment by Marcelinna** [comment]
For every status we stay, and delete service is still visible in backoffice

**Comment by AlicjaSwierad** [comment]
The most important: we need **clear information for the user.** 

- If we return to the main page, a pop-up should be visible for a few seconds with the message "You have deleted the service". 
- If we stay on the page of the deleted service, a notification at the top should indicate that it has been deleted.

Secondly, @Marcelinna  has a point, **we need to stay consistent.** "Similar when changing status to unpublish, suspended."

I will look for research/sources on such redirects.

**Comment by Michal-Kolomanski** [comment]
> The most important: we need **clear information for the user.**
> 
> * If we return to the main page, a pop-up should be visible for a few seconds with the message "You have deleted the service".
> * If we stay on the page of the deleted service, a notification at the top should indicate that it has been deleted.
> 
> Secondly, @Marcelinna has a point, **we need to stay consistent.** "Similar when changing status to unpublish, suspended."
> 
> I will look for research/sources on such redirects.

I agree that regardless of where the user will end up (after the delete action) some effect should be triggered. 
In terms of whether a user should be redirected or not, I will politely shut up and listen because it is purely a UX decision :D

**Comment by AlicjaSwierad** [comment]
In my opinion, we should stay on the page and add a button to the notifications to give users quick access to the main page

like this:
![image](https://github.com/user-attachments/assets/c9b242e4-9a2d-4306-acbf-911931a10e62)

**Comment by Michal-Kolomanski** [comment]
> In my opinion, we should stay on the page and add a button to the notifications to give users quick access to the main page
> 
> like this: ![image](https://private-user-images.githubusercontent.com/145433362/353812460-c9b242e4-9a2d-4306-acbf-911931a10e62.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI0MjU2MzksIm5iZiI6MTcyMjQyNTMzOSwicGF0aCI6Ii8xNDU0MzMzNjIvMzUzODEyNDYwLWM5YjI0MmU0LTlhMmQtNDMwNi1hY2JmLTkxMTkzMWExMGU2Mi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzMxJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDczMVQxMTI4NTlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05MjVjNjhlZGRhNjVjYjk0ZDVlNmU5OTM1ZmFkNWMyY2E2NmRiMmE0YzE2NGZlMjBkZDE3YWQ1OGE2Y2JlNGU4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.RFCro4LZ-dUxHGrgxi84haLaJIO6DpYksShKv1ApNEE)

For me it totally makes sense! 👏

**Comment by Michal-Kolomanski** [comment]
Design is finished? Could you please create a dev ticket with that figma?

**Comment by AlicjaSwierad** [comment]
The design was a sketch :P but thx :) I will design a delete confirmation pop-up (styling - front-end only) and create a ticket with these improvements.

**Comment by Michal-Kolomanski** [comment]
> The design was a sketch :P but thx :) I will design a delete confirmation pop-up (styling - front-end only) and create a ticket with these improvements.

thanks!

**Comment by AlicjaSwierad** [comment]
Figma with popup and go to the services page: 

https://www.figma.com/design/AzGImZhLPnUtKMrYsdKc75/Providers-HUb?node-id=8554-18758&t=nR03fL6zaMKWYWDV-1
